github-webhooks/validators/fixupCommits.js

```diff
@@ -15,7 +15,7 @@ const FixupValidator = function() {
             .then((commits) => {
                 const messages = commits.map((a) => a.commit.message);
                 const fixups = messages
-                    .filter((message) => message.startsWith('fixup!') || message.startsWith('squash!'));
+                    .filter((message) => message.startsWith('fixup!') || message.startsWith('squash!') || message.startsWith('amend!'));
```

Comment:
Come to think of it, `const hasFixups = !!messages.find((message) => message.startsWith('fixup!') || message.startsWith('squash!') || message.startsWith('amend!'))` would be slightly more efficient, but that's probably unnecessary optimization.